probe_scraper/parsers/events.py

```diff
@@ -60,7 +60,7 @@ def parse(self, filenames, version=None):
         # Events.yaml had a format change in 53, see bug 1329620.
         # We don't have important event usage yet, so lets skip
         # backwards compatibility for now.
-        if version and int(version) < 53:
+        if version and version < 53:
```

Comment:
Why are we removing `int(version)` ?

Reply:
One of the changes I made was to make version an integer until it needs to be cast to a string later in the job. That simplified a few of the instances of version comparison.

Reply:
Same as below, version is now an int.

Reply:
I see, thank you :)